### PR TITLE
Rename PreservedCopy to CompleteMoab

### LINKED ISSUE #980: rename PreservedCopy to CompleteMoab (blocked by #979)
rename model as well as underlying db table, and any relevant columns and indexes

### PR DESCRIPTION
**Pull request by tallenaz**
Fixes #980

### CODE REVIEW

**Comment by tallenaz** [comment]
I see a couple of cases I'll add commits for:

`pc_args --> cm_args`
`pres copy --> complete moab`

File: app/jobs/replicated_file_check_job.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Confirms existence of PreservedCopy on a zip endpoint.
+# Confirms existence of CompleteMoab on a zip endpoint.
 # Confirms the MD5 checksum matches in database and s3.
 # Usage info:
 # ReplicatedFileCheck.set(queue: :endpoint_check_us_west_2).perform_later(pc)
```
Comment (SaravShah) [inline app/jobs/replicated_file_check_job.rb]: pc to cm here?

File: app/lib/audit/catalog_to_moab.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@ def self.check_version_on_dir(last_checked_b4_date, storage_dir, limit=Settings.
       # We can't use ActiveRecord's .find_each, because that'll disregard the order .least_recent_version_audit
       # specified.  so we use our own batch processing method, which does respect Relation order.
       pcs_to_audit_relation =
```
Comment (SaravShah) [inline app/lib/audit/catalog_to_moab.rb]: variable name here?

File: app/lib/audit/catalog_to_moab.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@ def self.check_version_on_dir(last_checked_b4_date, storage_dir, limit=Settings.
...
-        PreservedCopy.least_recent_version_audit(last_checked_b4_date).by_storage_location(storage_dir)
+        CompleteMoab.least_recent_version_audit(last_checked_b4_date).by_storage_location(storage_dir)
       ActiveRecordUtils.process_in_batches(pcs_to_audit_relation, limit) do |pc|
```
Comment (SaravShah) [inline app/lib/audit/catalog_to_moab.rb]: `cms_to_audit_relation, limit) do |cm|`

File: app/lib/audit/catalog_to_moab.rb [hunk 4]
```diff
@@ -14,7 +14,7 @@ def self.check_version_on_dir(last_checked_b4_date, storage_dir, limit=Settings.
...
         c2m = CatalogToMoab.new(pc, storage_dir)
```
Comment (SaravShah) [inline app/lib/audit/catalog_to_moab.rb]: `pc` to `cm`

File: app/lib/audit/checksum.rb [hunk 5]
```diff
@@ -27,10 +27,10 @@ def self.validate_disk_all_storage_roots
 
     def self.validate_druid(druid)
       logger.info "#{Time.now.utc.iso8601} CV validate_druid starting for #{druid}"
-      pres_copies = PreservedCopy.by_druid(druid)
-      logger.debug("Found #{pres_copies.size} preserved copies.")
+      complete_moabs = CompleteMoab.by_druid(druid)
+      logger.debug("Found #{complete_moabs.size} complete moabs.")
       checksum_results_lists = []
-      pres_copies.each do |pc|
+      complete_moabs.each do |pc|
```
Comment (SaravShah) [inline app/lib/audit/checksum.rb]: `do |cm|`

**Comment by jmartin-sul** [comment]
last of the stray pres copy stuff i found:
```shell
$ git grep -- 'pres.*copy' ':(exclude)db/migrate/*'
app/models/complete_moab.rb:  # TODO: Port over statuses to archive pres_copy model
app/models/preserved_object.rb:  # This should implement the start of the replication process if status is unreplicated for an archival pres_copy
app/models/preserved_object.rb:  # Compare last_existence_check (from archive pres_copy) with archive TTL when checking the archival pres_copy status
db/README.md:* A `CompleteMoab` represents a physical copy of a `PreservedObject`, e.g. an "online" moab stored on premesis and accessed via NFS mount, an "archive" copy sitting on a cloud endpoint (manipulable via REST calls to a 3rd party cloud service), etc.
spec/jobs/replicated_file_check_job_spec.rb:      it "updates preserved copy status" do
spec/jobs/replicated_file_check_job_spec.rb:    it 'updates preserved copy status' do
spec/jobs/replicated_file_check_job_spec.rb:  context "preserved copy has status 'unreplicated'" do
```

didn't see any `pc` stuff left when i grepped the version of this branch that got merged.

**Pull request merged**